tests/experimental/callbacks/test_llama_index_trace_callback_handler.py

```diff
@@ -30,7 +30,7 @@ def test_callback_llm(mock_service_context: ServiceContext) -> None:
     response = query_engine.query(question)
     # Just check that the callback handler is called using the patched LLM
     assert response.response == "LLM predict"
-    spans = callback_handler._tracer.span_buffer
+    spans = callback_handler.get_spans()
```

Comment:
`get_spans` can be iterator
```suggestion
    spans = list(callback_handler.get_spans())
```

Reply:
Nice, TIL